apps/camera/js/lib/camera/focus.js

```diff
@@ -81,6 +81,7 @@ Focus.prototype.configureFocusModes = function(mode) {
   this.faceDetection =
     faceDetectionUserEnabled && faceDetectionSupported && mode === 'picture';
   this.mozCamera.addEventListener('focus', this.onAutoFocusStateChange);
+  this.updateFocusState('undefined');
```

Comment:
I don't like the idea of just inventing a new state named 'undefined'.  If you're trying to set the focus state back to its uninitialized value as it is when the app first starts, you could just use `undefined` instead of `'undefined'`.

Also, do you really want to call updateFocusState() here, knowing that it might have side effects other than resetting the state, or would it be better to just set `this.focusState = undefined;` directly here?

I don't actually know the answer, and am going to pass this review on to someone who knows the code better.